src/studio/src/designer/frontend/app-development/features/dataModeling/containers/DataModelingContainer.tsx

```diff
@@ -67,14 +65,32 @@ export default function DataModelingContainer(props: IDataModelingContainerProps
     setCreateButtonAnchor(event.currentTarget);
   };
   const onNewModelNameChanged = (e: any) => {
-    setNewModelName(e.target.value);
+    const name = e.target.value;
+    if (!name.match(/^[a-z][a-zA-Z0-9_.\-æÆøØåÅ ]*$/)) {
+      setNameError('Invalid name');
+    } else {
+      setNameError('');
+      setNewModelName(name);
+    }
+  };
+
+  const onNameFieldKeyDown = (e: any) => {
+    if (e.keyCode === 13 && !nameError) {
```

Comment:
Maybe this logic could be part of the Altinn input field?
That it returns if there is a confirm-button, the keyCode is 13 and there is a value-error.

Reply:
done